osxcollector/output_filters/opendns/api.py

```diff
@@ -56,18 +57,56 @@ def categorization(self, domains):
             A dict of {domain: categorization_result}
         """
         url_path = 'domains/categorization/?showLabels'
-        response = self._requests.multi_post(self._to_url(url_path), data=simplejson.dumps(domains))
-        response = response[0]
+        all_responses = {}
+
+        if self._cache:
+            api_name = 'opendns-categorization'
```

Comment:
What do you think about defining this static string near the top of this class? e.g.
...
BASE_URL = 'https://investigate.api.opendns.com/'
OPENDNS-CATEGORIZATION = 'opendns-categorization'
...
same goes for other similar static strings like 'opendns-security', 'opendns-cooccurrences', 'shadowserver-bin-test', 'virustotal-file-reports', etc.